source/content/advanced-global-cdn.md

```diff
@@ -1,75 +1,127 @@
 
-### How do I get Advanced Global CDN?
+### SOC 2 Type 2 Compliance
 
-[Contact us](https://pantheon.io/contact?docs) to let us know when you're ready to unlock the full potential of the Pantheon Global CDN. Our Professional Services team will talk through your needs and help determine the best configuration for you.
+ SOC 2<Popover title="SOC" content="System and Organization Controls (SOC) is a suite of audit reports defined by the American Institute of Certified Public Accountants (AICPA)." /> Type 2 compliance provides third-party assurance to our customers about the conformance of Pantheon’s information security system with industry standards. Our SOC 2 compliance covers the Security and Availability [Trust Services Criteria](https://www.aicpa.org/interestareas/frc/assuranceadvisoryservices/trustdataintegritytaskforce.html).
 
-## Features
+### IP Allowlisting & Blocklisting
+
+ Granular access control down to the IP address level helps protect intellectual property and secure development environments.
+
+- Blocklist - prevent users with specific IPs or in IP ranges from accessing your site.
 
-- **Full-page & asset caching**
+- Allowlist - only allow users with specific IPs or in IP ranges to access your site, blocking everyone else.
 
-  Unlike most CDNs that only handle images and other static assets, full-page technology lets you cache entire pages and assets at the edge.
-- **Layer 3 & 4 DDOS protection & mitigation**
-- **SOC2 compliance**
-- **IP allowlisting & blocklisting**
-  - **Blocklist** - prevent users with specific IPs or in IP ranges from accessing your site.
-  - **Allowlist** - only allow users with specific IPs or in IP ranges to access your site, blocking everyone else.
-- **Token authentication / JWT (JSON Web Token)**
+### Token Authentication / JWT (JSON Web Token)
 
   Use the AGCDN Edge to generate JSON Web Tokens<Popover title="JSON Web Tokens" content="A JSON Web Tokens is an Internet standard for creating compact, encrypted JSON-based access tokens that assert some number of claims, such as 'logged in as admin'." /> to help build your custom API.
-- **Custom error & maintenance page rules**
-- **Geolocation based edge control**
 
-  Take action on the edge based on the physical location of the user (eg. blocking or redirecting based on country) or send back location information to your application for more custom control.
-- **Device detection and edge control**
-- **Persistent Uptime**
+### Custom Error & Maintenance Page Rules
+
+ Response code messages enable you to create custom error pages (e.g. 404 and 503 errors) for a tailored end-user experience.
+
+### Geolocation-based Actions
+
+  Enable actions based on the physical location of your user (eg. blocking or redirecting based on country) or send location information to your application for more custom control.
```

Comment:
I would also include here that sending location info to the application will almost always be much faster than fetching that information from a third-party API, which is currently what most people are doing for geolocation.